scripts/simulator.sh

```diff
@@ -25,8 +25,9 @@ if [ $EXIT_CODE -ne 0 ]; then
   exit 1
 fi
 
-if which npx >/dev/null; then
-    npx expo run:ios --scheme $1 --configuration $2 ${@:3}
+# check if expo is installed as devDependency
+if [ -d "./node_modules/expo" ]; then
+    expo run:ios --scheme $1 --configuration $2 ${@:3}
 else
-    react-native run-ios --scheme $1 --configuration $2 ${@:3}
+    react-native run-ios --scheme $1 --mode $2 ${@:3}
```

Comment:
this it's a new change introduced in `react-native run` command

Reply:
You should also update this flag in all `android:{env}` scripts from `package.json`.